src/action.ts

```diff
@@ -44,14 +43,16 @@ export const authenticate = async (client_id: string, client_secret: string): Pr
     const url = new URL('/oauth/token', AUTH_URL)
 
     try {
-        const response = await axios.post(url.href, {
-            grant_type: 'client_credentials',
-            client_id,
-            client_secret,
-            audience: 'https://api.devcycle.com/',
+        const formData  = new FormData();
```

Comment:
why FormData?

Reply:
Because that's how the request is being sent/should be sent as